packages/test/test-service-load/src/loadTestDataStore.ts

```diff
@@ -410,7 +410,7 @@ export class LoadTestDataStoreModel {
 	}
 
 	public async volunteerForTask() {
-		if (this.runtime.disposed) {
+		if (this.isPermanentlyDisconnected()) {
```

Comment:
I wonder why this code (and any similar code in this PR) should care? I.e. if the error is hit, container is in closed state, and harness (driving all the runners) should dispose container. Is this piece of code that is simply missing??